### ISSUE TITLE
Multi-line captions

### ISSUE THREAD

**Issue by LPrenzel** [emareg/paper-checker#24]
Hey,
in multiline captions the checker complains about missing dots.

```
\caption{
    abcdef hijkl.
}
```

It insteads recommends:
```
\caption{
    abcdef hijkl.
.}
```

Also, it recommends `an unanimous`, although `a unanimous` is correct. (It begins with a "y" sound -> `a year`).

Thanks!

**Comment by emareg** [comment]
"an unanimous" is a known bug, I have added it to the testfile under false positives. The caption should work correctly now but the display of multiline suggestions is also somehow flawed...
